elasticai/creator/vhdl/language.py

```diff
@@ -215,6 +310,39 @@ def _wrap_in_IS_END_block(
     yield f"{Keywords.END.value} {block_type.value} {block_identifier};"
 
 
+def _wrap_in_ARCHITECTURE_block(
```

Comment:
In case the `_wrap_in*` functions are only used in the corresponding classes i think we should put them there or just inline the code in the corresponding `__call__` functions